pytarallo/Tarallo.py

```diff
@@ -98,41 +102,92 @@ def status(self):
         """
         try:
             return self.get('/v2/session').status_code
-        except AuthenticationError:
+        except Errors.AuthenticationError:
             return self.response.status_code
 
     def get_item(self, code, depth_limit=None):
-        """This method returns an Item instance"""
-        url = '/v2/items/' + self.urlencode(code)
+        """This method returns an Item instance received from the server"""
+        url = '/v2/items/' + self.urlencode(code) + '?separate' # try an Item without product6
```

Comment:
Maybe we should use fstrings everywhere to improve readability. See also following lines.

Reply:
Agree, fstrings are just so much more readable than any other form of string concatenation